versions/draft-3/SPEC.md

```diff
@@ -754,9 +743,19 @@ task test {
 }
 ```
 
-In this case `flags` within the `${`...`}` is an expression.  The `$expression` can also be more complex, like a function call: `write_lines(some_array_value)`
+In this case `flags` within the `${...}` is a variable lookup expression.  The expression can also be more complex, like a function call: `write_lines(some_array_value)`
+
+Here is the same example using the `command <<<` style:
+```wdl
+task test {
+  String flags
+  command <<<
+    ps ~{flags}
+  >>>
+}
+```
 
-> **NOTE**: the `$expression` in this context can only evaluate to a primitive type (e.g. not `Array`, `Map`, or `Object`).  The only exception to this rule is when `sep` is specified as one of the `$var_option` fields
+> **NOTE**: the expression in this context must be interpolated as a string in the command. This is immediately possible for WDL primitive types (e.g. not `Array`, `Map`, or `Object`). To place an array into the command block a separater character must be specified using `sep` (eg `${sep=", " int_array}`).
```

Comment:
I Am Not a WDL Language Lawyer but I wonder if there's a benefit to distinguish 'string interpolation' with 'string replacement' in the text. It sounds like the same thing to me, and 'replace' seems like a simpler term.